core/src/main/java/com/graphhopper/routing/ev/Country.java

```diff
@@ -269,4 +272,14 @@ public String getTwoLetterCode() {
     public static EnumEncodedValue<Country> create() {
         return new EnumEncodedValue<>(Country.KEY, Country.class);
     }
+
+    private static final Map<String, Country> map = new ConcurrentHashMap<>();
+
+    static {
+        for (Country c : Country.values()) map.put(c.getTwoLetterCode(), c);
```

Comment:
We could also add case-insensitive lookups while we're at it

```suggestion
        for (Country c : Country.values()) {
            map.put(c.getTwoLetterCode(), c);
            map.put(c.getTwoLetterCode().toLowerCase(Locale.ROOT), c);
        }
```